### DISCUSSION TITLE
Adding a tag from body text

### DISCUSSION THREAD

**Discussion by serrq** [spacecowboy/Feeder#310] [Ideas]
![1000113745](https://github.com/spacecowboy/Feeder/assets/113377431/41683478-cc01-4405-ba83-3f1b1f5a3016)

**Comment by spacecowboy** [comment]
Not sure what you mean by "adding a tag"

**Reply by serrq** [reply]
Adding a tag in a tags system. So you have to create a tags system before.

**Reply by spacecowboy** [reply]
I don't think this feature makes any sense since you can only have one tag per feed

**Reply by serrq** [reply]
I respect your point of view. Please allows me to explain mine.

Each article can having multiple key words: so my idea for the article above was `ripple` `xrp` `archax` `crypto` tags.

**Reply by spacecowboy** [reply]
I figured that. But in Feeder, a feed can only have a single tag. And an individual article, has none.

A tag in feeder is more like a folder.

**Reply by serrq** [reply]
And thus what is the solution that you propose? I didn't understand this part. Are you working on something similar or I am lost something in Feader features?

**Reply by spacecowboy** [reply]
There is no solution. I have not stated any intention to work on anything like this.

But you are free to do so. I'm happy to give advice on contributions.

**Reply by serrq** [reply]
Thanks dev for your reply. Unfortunately I am not a developer.

**Reply by spacecowboy** [reply]
No worries not-dev!
